org/pr/release-notes.ts

```diff
@@ -0,0 +1,37 @@
+import {warn, danger} from "danger";
+
+export default async () => {
+    const githubLabels = danger.github.issue.labels;
+    const modifiedFiles = danger.git.modified_files;
+    const pr = danger.github.pr;
+
+    // List of release notes files
+    const releaseNotesFiles: string[] = [
+        "RELEASE-NOTES.txt"
+    ];
+
+    // Skip if not on a release branch 
+    if (!pr.base.ref.startsWith("release/")) {
+        return;
+    }
+
+    // Skip for PR labeled as "Releases"
+    if (githubLabels.length != 0) {
+        const releases = githubLabels.some(label => label.name.includes("Releases"));
+        if (releases) {
+            return;
+        }
+    }
```

Comment:
Mmmh maybe I'm not completely up-to-date with when we use this Releases label and when not (I thought it was equally applied both for PRs targetting the release branch – like fixes made during the code freeze – and for PRs merging the release branch (back into develop after new betas, back into develop + trunk after store submission)?

Or should this label only be used for the PRs from release to develop and trunk that are happening after store submission, but should not be used for PRs targetting the current code-frozen release?

I fear that people might easily add that "Releases" label to PRs fixing code-freeze-bugs and thus targetting the `release/*` branch anyway, which would make this test to be skipped if so; not sure why we would want to test the label anyway if we already test that the target branch is `release/*` 🤔

Reply:
My understanding of the "Releases" label is that we use it to mark the PRs that merge beta and finalized versions back into `develop` and `trunk`.

That helps with another warning, the one for the code generated [`Localizable.strings`](https://github.com/Automattic/peril-settings/blob/685783686a07f5a09ee8d65acbf96335f7b0b146/org/pr/ios-macos.ts#L32-L45).

Reply:
> I fear that people might easily add that "Releases" label to PRs fixing code-freeze-bug

Working with the Simplenote team, I've never seen this pattern. 

That doesn't mean that it wouldn't be possible, though. If we're concerned about it, maybe the label could have a more explicit name, like "Release Merge".

Reply:
TBH I think I'm guilty of having used that `Releases` label that way (i.e. used it also for PRs merging fixes into the code-frozen `release/*` branch), not really knowing what its real definition/purpose was. Not sure if "Release Merge" would help disambiguate.

This could actually be another Peril check: ensure that if the Releases label is set, the PR is from `release/*` and targeting either `develop` or `trunk`, but not a PR targeting `release/*` itself.
Bonus question: are there PRs from `release/*` to `develop` or `trunk` that we _don't_ label with the `Releases` label? (maybe the initial one immediately after code freeze?) if we label all `release/* -> (develop|trunk)` with the label maybe we can automate that label (with Peril? can it _set_ labels?)

Reply:
> This could actually be another Peril check: ensure that if the Releases label is set, the PR is from `release/*` and targeting either `develop` or `trunk`, but not a PR targeting `release/*` itself.

Great idea!

Reply:
> are there PRs from `release/*` to `develop` or `trunk` that we _don't_ label with the Releases label? (maybe the initial one immediately after code freeze?) if we label all `release/* -> (develop|trunk)` with the label maybe we can automate that label

I can't think of any occasion in which I didn't have to use Release. Every merge of the release branch in `develop` corresponds to a new build that's being distributed, as far as I know.

> (with Peril? can it set labels?)

It can. Peril has access to an authenticated GitHub API wrapper via Danger.